cfpb/consumerfinance.gov
# Wagtail: Rename Chinese to Chinese (Traditional)

Follow-up to https://github.com/cfpb/consumerfinance.gov/pull/6475 to rename `Chinese` to `Chinese (Traditional)` in language drop-down.

## Changes

- Rename `Chinese` to `Chinese (Traditional)` language drop-down.


## How to test this PR

1. Pull branch, run migrations.
2. Edit or create a page in wagtail and go to the configuration tab and see that the language drop-down now has Chinese (Traditional) as an option.


## Screenshots

<img width="732" alt="Screen Shot 2021-06-11 at 10 39 43 AM" src="https://user-images.githubusercontent.com/704760/121704041-6223f080-caa1-11eb-9241-54269e9abeaa.png">
